### DISCUSSION TITLE
Need guidance in contributing to Open3D

### DISCUSSION THREAD

**Discussion by Shazam213** [isl-org/Open3D#5880] [General]
Hello everyone i am a sophomore and have previously worked with 3d data with one of my projects being terrain reconstruction of data collected from a drone. Now i am interested in contributing to open3d.  I have build it from source and also read the contributors guide but now i am confused from where should i start. Can anyone please guide me.

**Comment by ssheorey** [comment]
Hey @Shazam213 the next place would be to start looking at the list of issues. Look for something that interests you, explore the code and reproduce the issue, ask the issue creator for more information if necessary, and then submit a pull request to resolve it.
